projects/ngx-openlayers/src/lib/view.component.ts

```diff
@@ -42,12 +42,19 @@ export class ViewComponent implements OnInit, OnChanges, OnDestroy {
   @Input()
   zoomAnimation = false;
 
-  constructor(private host: MapComponent) {}
+  @Output()
+  onZoomChange: EventEmitter<ObjectEvent>;
+
+  constructor(private host: MapComponent) {
+    this.onZoomChange = new EventEmitter<ObjectEvent>();
```

Comment:
Seems more angular idiomatic to init this on the property

Reply:
Indeed @davinkevin, but if we do this here we have to do it everywhere. I think we should open a specific issue for that.

Reply:
as for the other discution I did update this one but we should open an issue to fix it everywhere else